lms/views/api/sync.py

```diff
@@ -8,10 +7,6 @@
 
 
 class APISyncSchema(PyramidRequestSchema):
-    class LMS(Schema):
```

Comment:
This is no longer required, we are not sending this information from JSConfig anymore.